docs/source/getting_started/writing_code.ipynb

```diff
@@ -115,6 +88,24 @@
     "\n",
     "Much of the `maap.py` functionality is documented in the [Technical Tutorials section](../technical_tutorials.rst) and in-context in the [Science Tutorials](../science_examples.rst). The [maap-py Github page](https://github.com/MAAP-Project/maap-py) has additional usage documentation.\n"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "plaintext"
```

Comment:
@rtapella Do you want me to change this to a markdown cell so it looks cleaner? 
<img width="699" alt="Screenshot 2024-06-06 at 6 47 06 AM" src="https://github.com/MAAP-Project/maap-documentation/assets/114033465/a95649b5-cad0-4225-9a90-b1bd7f186378">

Reply:
Yes please… I thought this was rendering properly when I did a local build… thank you

Reply:
Done!